tools/ui-components/package.json

```diff
@@ -45,6 +46,7 @@
     "babel-loader": "8.2.3",
     "babel-plugin-transform-react-remove-prop-types": "0.4.24",
     "cross-env": "7.0.3",
+    "cssnano": "^5.0.14",
```

Comment:
I think we need `postcss-cli` and some postcss config update in order for this to work? Ref: https://cssnano.co/docs/getting-started/#using-postcss-cli

Though I'm not really sure here to be honest (I'm not familiar with the plugin and still catching up 😅)

Reply:
Right. The postcss config has to change, I will take it out for now.